src/anemoi/models/preprocessing/__init__.py

```diff
@@ -47,8 +63,10 @@ def __init__(
         self.data_indices = data_indices
 
     def _process_config(self, config):
```

Comment:
Can you put a comment to explain why these keys are handled specially?

Reply:
ok